src/main/java/edu/kis/powp/jobs2d/features/Transformations/TransformationDecorator.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.powp.jobs2d.features.Transformations;
+
+public class TransformationDecorator implements Transformation{
```

Comment:
again, what is it for? I do not see any gain from this decorator? 
For example, it would be nice to have a possibility to use multiple transformations at once.